docs/CHANGELOG.asciidoc

```diff
@@ -32,9 +32,10 @@
 
 === Bug Fixes
 
-Correct query times for model plot and forecast in the bucket to match the times we assign
-the samples we add to the model for each bucket. For long bucket lengths, this could result
-in apparently shifted model plot with respect to the data and increased errors in forecasts. 
+Corrects query times for model bounds and forecasts in the bucket to match the
+times assigned to the samples that are added to the model for each bucket. For
+long bucket lengths, this fix could result in apparently shifted model bounds
```

Comment:
I think this line needs changing as it refers to the situation _before_ the fix.  One possibility would be to simply remove the word "fix".  Or if you want the word "fix" to appear somewhere, another way would be to completely flip the paragraph:

> Fixes a problem that could result in shifted model bounds and increased forecast errors with long bucket spans.  The change was to correct query times for model bounds and forecasts in the bucket to match the times assigned to the samples that are added to the model for each bucket.  (See {ml-pull}327[#327].)

Reply:
or possibly "fix" -> "bug", but I think I maybe prefer Dave's suggestion